tests/Standards/AbstractSniffUnitTest.php

```diff
@@ -197,8 +197,10 @@ final public function testSniff()
                         $fixedFilename     = basename($fixedFile);
                         $failureMessages[] = "Fixed version of $filename does not match expected version in $fixedFilename; the diff is\n$diff";
                     }
+                } else {
+                    $this->addWarning("Unable to verify auto-fixer results. File not found: $fixedFile");
```

Comment:
I wonder if this message could be made more specific as it could now give the impression that _all_ test case files for sniffs containing fixers are expected to have a `.fixed` file, even when the fixers should/will not run for the test case file.

What about something like:
_"Missing fixed version of $filename to verify the accuracy of fixes, while the sniff is making fixes against the test case file"_

Reply:
I don't understand the scenario that you're referring to. When would the test suite _not_ need a ".fixed" file for a fixer?
I can understand when a test file only contains errors which cannot be fixed automatically, but that code path does not reach here.

I've updated the text as suggested.

Reply:
> I don't understand the scenario that you're referring to. When would the test suite not need a ".fixed" file for a fixer?
I can understand when a test file only contains errors which cannot be fixed automatically, but that code path does not reach here.

I agree that the code path without fixable errors would not throw the notification, but the person _running_ the test suite will not necessarily know that, which is why I suggested the slightly different text.